src/clientmetricreport/DefaultClientMetricReport.ts

```diff
@@ -329,6 +352,90 @@ export default class DefaultClientMetricReport implements ClientMetricReport {
     }
   }
 
+  /**
+   *  media Stream metrics
+   */
+
+  readonly observableVideoMetricSpec: {
+    [id: string]: {
+      source: string;
+      media?: MediaType;
+      dir?: Direction;
+    };
+  } = {
+    videoUpstreamBitrate: {
+      source: 'bytesSent',
+      media: MediaType.VIDEO,
+      dir: Direction.UPSTREAM,
+    },
+    videoUpstreamPacketsSent: {
+      source: 'packetsSent',
+      media: MediaType.VIDEO,
+      dir: Direction.UPSTREAM,
+    },
+    videoUpstreamFramesEncodedPerSecond: {
+      source: 'framesEncoded',
+      media: MediaType.VIDEO,
+      dir: Direction.UPSTREAM,
+    },
+    videoUpstreamGoogFrameHeight: {
+      source: 'googFrameHeightSent',
+      media: MediaType.VIDEO,
+      dir: Direction.UPSTREAM,
+    },
+    videoUpstreamGoogFrameWidth: {
+      source: 'googFrameWidthSent',
+      media: MediaType.VIDEO,
+      dir: Direction.UPSTREAM,
+    },
+    videoDownstreamBitrate: {
+      source: 'bytesReceived',
+      media: MediaType.VIDEO,
+      dir: Direction.DOWNSTREAM,
+    },
+    videoDownstreamPacketLossPercent: {
+      source: 'packetsLost',
+      media: MediaType.VIDEO,
+      dir: Direction.DOWNSTREAM,
+    },
+    videoDownstreamFramesDecodedPerSecond: {
+      source: 'framesDecoded',
+      media: MediaType.VIDEO,
+      dir: Direction.DOWNSTREAM,
+    },
+    videoDownstreamGoogFrameHeight: {
+      source: 'googFrameHeightReceived',
+      media: MediaType.VIDEO,
+      dir: Direction.DOWNSTREAM,
+    },
+    videoDownstreamGoogFrameWidth: {
```

Comment:
If you do not use legacyStats, why do you add googleFrameWidth here?

Reply:
It is not necessary to have this googleFrameWidth. I will remove it.